_opensearch/query-dsl/full-text.md

```diff
@@ -402,26 +393,87 @@ GET _search
+
+To use the analyzer when you map an index, specify the value within your query. For example, to map your index with the French language analyzer, enter the analyzer field and specify the `french` value for the analyzer field:
+
+```json
+ ` "analyzer": "french"`
+ ```
+
+#### Sample Request
+
+The following query maps an index with the language analyzer set to French:
+
+```json
+PUT my-index-000001
+
+{
+  "mappings": {
+    "properties": {
+      "text": { 
+        "type": "text",
+        "fields": {
+          "french": { 
+            "type":     "text",
+            "analyzer": "french"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+<!-- TO do: each of the options needs its own section with an example. Convert table to individual sections, and then give a streamlined list with valid values. -->
 
-## Options
+## Optional query fields
+
+You can filter your query results by using some of the optional query fields such as wildcards, analyzers, fuzzy query fields, and synonyms.
+
+### Use wildcards
 
 Option | Valid values | Description
 :--- | :--- | :---
-`allow_leading_wildcard` | Boolean | Whether `*` and `?` are allowed as the first character of a search term. The default is true.
+`allow_leading_wildcard` | Boolean | Whether `*` and `?` are allowed as the first character of a search term. The default is true. Boolean values can be either `true` or `false`.
 `analyze_wildcard` | Boolean | Whether OpenSearch should attempt to analyze wildcard terms. Some analyzers do a poor job at this task, so the default is false.
-`analyzer` | `standard, simple, whitespace, stop, keyword, pattern, <language>, fingerprint` | The analyzer you want to use for the query. Different analyzers have different character filters, tokenizers, and token filters. The `stop` analyzer, for example, removes stop words (e.g. "an," "but," "this") from the query string.
-`auto_generate_synonyms_phrase_query` | Boolean | A value of true (default) automatically generates [phrase queries](https://lucene.apache.org/core/8_9_0/core/org/apache/lucene/search/PhraseQuery.html) for multi-term synonyms. For example, if you have the synonym `"ba, batting average"` and search for "ba," OpenSearch searches for `ba OR "batting average"` (if this option is true) or `ba OR (batting AND average)` (if this option is false).
+
+### Use built-in analyzers
+
+`analyzer` | `standard, simple, whitespace, stop, keyword, pattern, language fingerprint` | The analyzer you want to use for the query. Different analyzers have different character filters, tokenizers, and token filters. The `stop` analyzer, for example, removes stop words (e.g. "an," "but," "this") from the query string. For a full list of acceptable language values, see [Convert text with analyzers](#convert-text-with-analyzers) on this page.
```

Comment:
In the Valid values column, should there be a comma after "language"? Add a comma after "e.g.".